sklearn/metrics/tests/test_pairwise.py

```diff
@@ -138,6 +139,20 @@ def test_pairwise_boolean_distance(metric):
             res[np.isnan(res)] = 0
             assert np.sum(res != 0) == 0
 
+    # non-boolean arrays are converted to boolean for boolean
+    # distance metrics with a data conversion warning
+    msg = ("Data was converted to boolean "
+           "for metric %s" % (metric))
+    with pytest.warns(DataConversionWarning,
+                      match=msg) as w:
+        pairwise_distances(X, metric=metric)
+    assert len(w) == 1
```

Comment:
Not need for this assert here, and you don't need `as w`

Reply:
done